### PR TITLE
Add IBM i specific change to support the build.

### PR DESCRIPTION
**Pull request by gaolibj**
Signed-off-by: gaoli <gaolibj@cn.ibm.com>

### CODE REVIEW

File: tools/hookgen/makefile [hunk 1]
```diff
@@ -31,6 +31,11 @@ OBJECTS := $(patsubst %.cpp,%$(OBJEXT), $(wildcard *.cpp))
 OBJECTS += pugixml$(OBJEXT)
 
 MODULE_INCLUDES += $(OMR_PUGIXML_DIR)
+ifeq ($(I5_VERSION),I5_V7R3)
+#__setjmp supported on pase 7.3, but doesn't support on AIX 7.1.1 which we run the build.
+    MODULE_LIBPATH += /osxpf/v7r2m0.cuep/bld/cmvc/pase.pgm/p5.cuep/lib
```
Comment (rwy0717) [inline tools/hookgen/makefile:36]:
```suggestion
  # __setjmp supported on pase 7.3, but doesn't support on AIX 7.1.1 which we run the build.
  MODULE_LIBPATH += /osxpf/v7r2m0.cuep/bld/cmvc/pase.pgm/p5.cuep/lib
```

File: tools/hookgen/makefile [hunk 2]
```diff
@@ -31,6 +31,11 @@ OBJECTS := $(patsubst %.cpp,%$(OBJEXT), $(wildcard *.cpp))
...
+endif
+
```
Comment (rwy0717) [inline tools/hookgen/makefile:38]:
```suggestion
```

**Comment by rwy0717** [comment]
The two suggested changes are missing the signed-off-by line. The easiest fix is to squash those suggested changes into the original commit (which had the correct signed-off-by line). If you want, I can do this for you when I merge this PR. Let me know.

@genie-omr build all

**Comment by rwy0717** [comment]
@genie-omr build win

**Comment by gaolibj** [comment]
@rwy0717 Please help to squash those suggested changes into the original commit when you merge this PR. Thanks.

**Pull request merged**